### ISSUE TITLE
Config file location is not configurable

### ISSUE THREAD

**Issue by otoinsa** [brunch/brunch#1509]
### Description

First of all - I like brunch very much :)
Finally a tool that does it with pretty much no headache :D That's quite the achievement.

I usually keeep all my config files in a config/  folder; the cli tools I use
allow to pass a -c flag usually, where I can define the path of my config.

I think this would be a great change, as it would allow to clean up the code directory even more. 🎱 
### Expected behavior

`brunch watch -c config/brunch-config.js`
Could instruct brunch of where the config location is.
### Actual behavior

`brunch watch -c ./config/brunch-config.js`
won't work

`brunch watch -e ./config/brunch-config.js`
won't work either
### Environment
1. Brunch: 2.8.2
2. Node: 6.5.0
3. NPM: 3.10.3
4. Operating system: OS X 10.11.6
### `package.json` contents

```
{
  "name": "your-app",
  "description": "Description",
  "author": "Your Name",
  "version": "0.0.1",
  "repository": {
    "type": "git",
    "url": ""
  },
  "scripts": {
    "start": "brunch watch",
    "prod": "brunch build --production"
  },
  "dependencies": {
    "react": "~15.3.0",
    "react-dom": "~15.3.0"
  },
  "devDependencies": {
    "auto-reload-brunch": "^2.0.0",
    "babel-brunch": "~6.0.0",
    "babel-preset-es2015": "~6.9.0",
    "babel-preset-react": "~6.11.1",
    "brunch": "^2.4.0",
    "clean-css-brunch": "^2.0.0",
    "css-brunch": "^2.0.0",
    "javascript-brunch": "^2.0.0",
    "uglify-js-brunch": "^2.0.0"
  }
}

```
### brunch config contents

```
module.exports = {
  files: {
    javascripts: {
      joinTo: {
        'vendor.js': /^(?!app)/,
        'app.js': /^app/
      }
    },
    stylesheets: {joinTo: 'app.css'}
  },

  plugins: {
    babel: {presets: ['es2015', 'react']}
  }
};

```

**Comment by valmirjunior0088** [comment]
I would also very much like this feature, since I would also like to load my Brunch configuration file from a "/config" folder.

Perhaps a command line argument or a key in package.json to indicate the path?

**Comment by stelmakh** [comment]
@otoinsa @valmirjunior0088 Hi guys, just wanted to let you know that the feature is in master now and will be available as soon as the new version of Brunch is released. Thanks for your request!

**Comment by shvaikalesh** [comment]
Released in `2.10`.
